site/data/qasida/ya-nasim-al-ward.json

```diff
@@ -5,34 +5,61 @@
     },
     "verses": {
         "ara": [
-            ["يَا نَسِيمَ الوَرْدِ خَبِّرْ لِلرَشَا"],
-            ["لَم يَزِدْنِي الوِرْدُ إلَا عَطَشَا"],
+            ["لَا إِلَهَ إِلاَّ اللهْ اللهْ اللهْ"],
+            ["لَا إِلَهَ إِلَّا اللَّهُ جُدْ عَلَيْنَا"],
 
-            ["لِي حَبِيبٌ حُبُهُ وَسْطَ الحَشَا"],
-            ["إن يَشَأ يَمْشِي عَلَى خَدِّي مَشَا"],
+            ["يَا نَسِيْمَ الْوَرْدِ خَبِّرْ لِلرَّشَا"],
+            ["لَمْ يَزِدْنِي الْوِرْدُ إِلَّا عَطَشَا"],
 
-            ["قَولُهُ قَولِي وَقَولِي قَولُهُ"],
-            ["إن يَشَأ شِئتُ وَإن شِئتُ يَشَا"],
+            ["لِيْ حَبِيبٌ حُبُّهُ حَشْوُ الْحَشَا"],
```

Comment:
Note the main change here (which agrees with the recording I found previously) is
```diff
-- وَسْطَ
++ حَشْوُ
```
I'm not sure if the translation needs updating, but the I'm not an experto as I'm not sure about any further linguistic or metaphorical uses.

I think the generated one also has its own rhyming scheme hence the word choice it used `love fills my core`

Reply:
Actually the version I originally knew was حشو (like in Songs of Presence), but I put وسط because it was in Sh Ma'n's version and I also found versions of it in online recordings...

I prefer the حشو version as you've done it!

Reply:
I feel like in general Sh Ma'n prefers the Songs of Presence wording over other sources so maybe we can roll with it until told otherwise